### PR TITLE
TVB-2719 lint: Use same convention everywhere

### LINKED ISSUE #60: Lint: Use same convention everywhere

### PR DESCRIPTION
**Pull request by ayan-b**
Closes https://github.com/the-virtual-brain/tvb-gdist/issues/60

### CODE REVIEW

**Comment by ayan-b** [comment]
I used [black](https://github.com/psf/black) to format the code.

**Review by liadomide — COMMENTED** [review]
These changes look good for the Python part. How about the Cython part of this library? Is there a cython lint tool which we could apply?

**Comment by ayan-b** [comment]
Modified flake8 configuration to enable linting for cython files.

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[flake8]
+filename = *.py, *.pyx
+ignore = E501, W503
+per_file_ignores = gdist.pyx: E999, E402, E266, E255, E227, E226, E225
```
Comment (liadomide) [inline setup.cfg]: Would it be possible to copy with every ignored error code also the explanation for it (what we are ignoring).
e.g. `E501 #  Line too long (82 > 79 characters)` ?

Comment (liadomide) [reply setup.cfg]: Is  `E402 | Module level import not at top of file` ?
Usually I don't like the stile of an import in another place that the top, unless there is a good reason. Do we really need this exception here?

Comment (liadomide) [reply setup.cfg]: Also are these the ignores we really want:
`E225 | Missing whitespace around operator
E226 | Missing whitespace around arithmetic operator
E227 | Missing whitespace around bitwise or shift operator`
They would seem fitting to be obeyed, to me.

Comment (ayan-b) [reply setup.cfg]: > 
> 
> Also are these the ignores we really want:
> `E225 | Missing whitespace around operator E226 | Missing whitespace around arithmetic operator E227 | Missing whitespace around bitwise or shift operator`
> They would seem fitting to be obeyed, to me.

For `pyx` files, flake8 will complain for the pointers (treated as multiplication operator in python) and address-of operator (bitwise and operator in python).

> 
> 
> Is `E402 | Module level import not at top of file` ?
> Usually I don't like the stile of an import in another place that the top, unless there is a good reason. Do we really need this exception here?

Flake8 treats the `cimport`s as non-import code and thus complains about the subsequent imports as imports not being on the top.


> 
> 
> Would it be possible to copy with every ignored error code also the explanation for it (what we are ignoring).
> e.g. `E501 # Line too long (82 > 79 characters)` ?

Done.

**Review by liadomide — APPROVED** [review]

**Pull request merged**